packages/rpc/lib/src/utils/unix_rpc_client.dart

```diff
@@ -17,12 +18,23 @@ class UnixRPCClient extends ServerProxyBase {
     var socket = await Socket.connect(address, 0);
     socket.add(utf8.encode(package));
 
-    socket.listen((event) {
-      LogManager.getInstance.debug('Event received is ${utf8.decode(event)}');
-      completer.complete(utf8.decode(event));
-    },
+    socket.listen(
+        (event) {
+          LogManager.getInstance
+              .debug('Event received is ${utf8.decode(event)}');
+          Map<String, dynamic> eventMap = jsonDecode(utf8.decode(event));
+          if (eventMap.containsKey("error")) {
+            var error = eventMap["error"];
+            error = LNClientException(
```

Comment:
```suggestion
            var exception = LNClientException(
```

please keep python away :D